Extensions/xkit_preferences.js

```diff
@@ -2445,16 +2627,16 @@ XKit.extensions.xkit_preferences = new Object({
 
 		$("#xkit-open-credits").click(function() {
 
-				XKit.window.show("Credits",
-					'XKit uses <a href="www.jquery.com">jQuery and jQuery UI</a> by jQuery Foundation, '+
-					'<a href="https://github.com/timrwood/moment/">moment.js</a> by Tim Wood, '+
-					'<a href="http://code.drewwilson.com/entry/tiptip-jquery-plugin">TipTip</a> by Drew Wilson and '+
-					'<a href="http://jamesflorentino.github.io/nanoScrollerJS/">nanoScroll</a> by James Florentino. '+
-					'<br/><br/>'+
-					'XKit is written by <a href="http://www.studioxenix.com/">STUDIOXENIX</a>, a one-man entity.<br/><br/>'+
-					'All trademarks are the property of their respective owners.<br/>'+
-					'By using this software you are agreeing to <a href="http://www.xkit.info/eula">XKit EULA</a>.',
-				"info", '<div class="xkit-button default" id="xkit-close-message">OK</div>');
+			XKit.window.show("Credits",
+			'XKit uses <a href="www.jquery.com">jQuery and jQuery UI</a> by jQuery Foundation, '+
+			'<a href="https://github.com/timrwood/moment/">moment.js</a> by Tim Wood, '+
+			'<a href="http://code.drewwilson.com/entry/tiptip-jquery-plugin">TipTip</a> by Drew Wilson and '+
+			'<a href="http://jamesflorentino.github.io/nanoScrollerJS/">nanoScroll</a> by James Florentino. '+
+			'<br/><br/>'+
+			'XKit is written by <a href="http://www.studioxenix.com/">STUDIOXENIX</a>, a one-man entity.<br/><br/>'+
+			'All trademarks are the property of their respective owners.<br/>'+
+			'By using this software you are agreeing to <a href="http://www.xkit.info/eula">XKit EULA</a>.',
+			"info", '<div class="xkit-button default" id="xkit-close-message">OK</div>');
```

Comment:
> Do you want it to be resolved this badly?

Yes.  As bvtsang says, it introduces unnecessary code churn.  I don't particularly care what tools developers prefer to work with, but they shouldn't be making changes (especially automatic ones) to unrelated parts of the PR.

@bvtsang can you add styleguide info to #209?  It's a good point.